shards/modules/spatial/CMakeLists.txt

```diff
@@ -8,5 +8,5 @@
 #   REGISTER_SHARDS spatial
 #   EXPERIMENTAL)
 
-# target_link_libraries(shards-module-spatial gfx input shards-module-egui)
+# target_link_libraries(shards-module-spatial gfx-core input shards-module-egui)
```

Comment:
Remove? Or just disabled for now I guess?

Reply:
Was already disabled, just fixed the reference with find-replace